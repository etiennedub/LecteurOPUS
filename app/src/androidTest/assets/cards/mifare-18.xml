<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">04D5643D</List><List index="1">2A444980</List><List index="2">A748E000</List><List index="3">0000000F</List><List index="4">00000000</List><List index="5">30DF0523</List><List index="6">00000000</List><List index="7">44434F0F</List><List index="8">01C697A0</List><List index="9">7BC03505</List><List index="10">2C000000</List><List index="11">22286E72</List><List index="12">01C697A0</List><List index="13">7A06050B</List><List index="14">2E000000</List><List index="15">2028AF3E</List></Pages></Mifare>