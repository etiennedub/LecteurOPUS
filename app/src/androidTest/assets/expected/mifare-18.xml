<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="30/11/2016" Time="16:30" Transfert="true" OperatorId="20" BusId="3" OperatorName="RTC" BusName="LEBUS 4" /><List index="1" Date="30/11/2016" Time="16:16" Transfert="false" OperatorId="20" BusId="96" OperatorName="RTC" BusName="METROBUS 801" /></Trip><Contract><List index="0" Subscription="false" Valid="true" NbTicket="0" OperatorId="20" OperatorName="RTC" /></Contract></Card>